test/unit/TestPANOSDriver.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2016 Dravetech AB. All rights reserved.
+#
+# The contents of this file are licensed under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+"""Tests."""
+
+import unittest
+
+from napalm_panos import panos
+from napalm_base import exceptions
+
+
+class TestConfigPANOSDriver(unittest.TestCase):
+    """Group of tests that test Configuration related methods."""
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
You just need this method, the rest should be inherited. You can add extra if you want but don't override the existing tests. Check `napalm-ios`.